src/components/_classes/component/Component.js

```diff
@@ -876,10 +876,13 @@ export default class Component extends Element {
     return null;
   }
 
+  getFormattedAttribute(attr) {
+    return this.t(attr, { _userInput: true }).replace(/"/g, '&quot;');
+  }
```

Comment:
If attr value is null or undefined, the replace call will cause an error. I would prefer to add the check, if attr is false, just return an empty string